buildSrc/src/main/java/Deps.kt

```diff
@@ -10,6 +10,10 @@ object versions {
         const val kotlin = "1.4.10"
     }
 
+    object android {
+        const val databinding = "3.1.4"
```

Comment:
should match with the plugin version
https://maven.google.com/web/index.html#androidx.databinding:adapters

Reply:
Seems that module doesn't have databinding codegen, that's why has removed kapt from here :)